src/lib/ui/noise-sensor/NoiseSensorEventItem.tsx

```diff
@@ -1,16 +1,20 @@
+import { DateTime } from 'luxon'
 import type { Event } from 'seamapi'
 
-import { formatDateTime } from 'lib/dates.js'
 import { ClockIcon } from 'lib/icons/Clock.js'
 
+const ABBREVIATED_DATE_FORMAT = 'MMM d, yyyy'
+const TIME_FORMAT = 'h:mm a'
```

Comment:
Generally we [should avoid defining our own date format string](https://moment.github.io/luxon/#/formatting?id=consider-alternatives), instead use a built in format from luxon, or set it using this method: https://github.com/seamapi/react/blob/1221de8024782ab974d40a2a663bf870137c36c2/src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx#L247-L260

Reply:
Good point! I've replaced them with Luxon contacts.